packages/components/tests/dummy/app/templates/components/alerts.hbs

```diff
@@ -0,0 +1,228 @@
+{{page-title "Alert components"}}
+
+<h2 class="dummy-h2">Alerts</h2>
+
+<section>
+  <p class="dummy-paragraph">Reference:
+    <a href="https://docs.google.com/document/d/1MC0MTm8-0F26s6Y-gou_RsMj6rtKS6jC-eM7rmruqLY/edit#heading=h.gjdgxs">
+      Alert Components Requirement Document (CRD)
+    </a>
+  </p>
+  <p>Starting with Alert: "Global" and Alert: "Page or Inline". (Alert: "Toast" coming next.)</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3">Component API</h3>
+  <p class="dummy-paragraph">Here is the API for the component:</p>
+
+  <dl class="dummy-component-props">
+    <dt>type <code>enum</code></dt>
+    <dd>
+      <p>Has a border and a background color. Names
+        <code class="dummy-code">global</code>
+        and
+        <code class="dummy-code">page or inline</code>
+        are temporary and subject to change.</p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">global</li>
+        <li>page or inline</li>
+      </ol>
+    </dd>
+
+    <dt>isCompact <code>boolean</code></dt>
+    <dd>
+      <p>
+        <code class="dummy-code">isCompact</code>
+        is a variation on the
+        <code class="dummy-code">page or inline</code>
+        alert type ONLY. An
+        <code class="dummy-code">isCompact</code>
+        alert does NOT have a border. An
+        <code class="dummy-code">isCompact</code>
+        alert does NOT have a background color.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>true</li>
+        <li class="default">false</li>
+      </ol>
+    </dd>
+
+    <dt>intent <code>enum</code></dt>
+    <dd>
+      <p><code class="dummy-code">intent</code>
+        result in a default
+        <code class="dummy-code">icon</code>, which can be overriden.
+        <code class="dummy-code">intent</code>
+        chooses a background color, border,
+        <code class="dummy-code">titleText</code>
+        color (if applicable), and
+        <code class="dummy-code">bodyText</code>
+        color (if applicable).
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>highlight</li>
+        <li>success</li>
+        <li>warning</li>
+        <li>critical</li>
+      </ol>
+    </dd>
+
+    <dt>hasIcon <code>boolean</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">true</li>
+        <li>false</li>
+      </ol>
+    </dd>
+
+    <dt class="dummy-indent">icon <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Use this parameter to show an icon. Exists if
+        <code class="dummy-code">hasIcon=true</code>.
+      </p>
+      <p>Acceptable value:</p>
+      <p>Any
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+          Flight icon</a>
+        name.
+      </p>
+    </dd>
+
+    <dt>titleText <code>string</code></dt>
```

Comment:
@cveigt I was thinking that `titleText`, which I think you mentioned, is a better name than `headerText`, since it's not a semantic HTML h3 or something. Let me know what you think

Throughout, I tried to capture all the updates from the last time we met (Thurs?). Is just the docs for now. Def let me know what you think and if anything should change

***

Next steps for me:
- Show examples in code w/all the `if` conditionals, following the Figmas for first stab at styling
- Show examples in code where it's not just one Button and one Link (Standalone), for example it could yield something custom e.g. 2 Buttons and one Link (Standalone)
- Anything else that comes to your mind?
- Toast